themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,182 @@
+
+<!--more-->
+
+## Azure Container Instances
+
+[Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) (ACI) bills itself as an offering that lets you "easily run containers on Azure without managing servers."
+
+Launched in July 2017, it was arguably the first "serverless" container offering from the major cloud providers.
+
+### What's good about it?
+
+ACI is a Platform-as-a-Service (PaaS) type of product that is designed to remove the burden of launching and running containers.
+
+#### Simplicity
+
+Azure Container Instances has been designed from the ground up to be straightforward to use. It’s designed to get your container launched as quickly as possible, even including integrations with many of the tools you’re probably already familiar with if you’re using containers. One example is the integration with the [Docker CLI](https://docs.microsoft.com/en-gb/azure/container-instances/quickstart-docker-cli).
+
+In many cases, the only configuration options you need to pass to your ACI provisioning mechanism is the image you’re running and some sizing decisions, like how many CPUs and how much memory you need. There is some flexibility in this decision process, but having fewer required options means you’re not bogged down with complex architectural decisions when trying to get your container running.
+
+This lends itself particularly well to web server projects, especially at smaller scales. You can launch applications very quickly and easily by specifying the image you've built, and your application can be running in very short order.
+
+#### Start-Up Times
+
+ACI Containers start fast&mdash;often in a matter of seconds. This speed really helps the developer lifecycle and helps to create productivity when you're developing your application.
+
+#### Pricing
+
+ACI will only bill you for what you use. It offers no start-up costs (which goes great with the fast start times!) and per-second billing, meaning you only pay for what you use. The pricing is also extremely competitive against other cloud providers.
+
+#### The Other Stuff
+
+From a security perspective, ACI separates your container from others running in the same environment, meaning your application will have the same level of security as if it was running on a different virtual machine. This separation really helps to provide peace of mind and is yet another benefit of the serverless model&mdash;less infrastructure means less worries.
+
+As expected from Microsoft, ACI has excellent support for Windows containers and .NET platform applications, which is lacking around other cloud providers. If you need to run Windows containers, this support is a huge benefit.
+
+### What's not great about it?
+
+#### Scaling
+
+ACI supports customizing the amount of resources you can assign to a container, but it will only run one copy. There's no horizontal scaling available, so if you need more resources, you only have one option: scaling vertically.
```

Comment:
There is an option of creating N equal independent instances and configuring load balancing across them, but it's very manual and awkward. Some people do that though.